lib/components/table.vue

```diff
@@ -407,14 +413,22 @@ export default {
         });
     },
     computed: {
+        computedDark() {
```

Comment:
At carousel-slide more laconic deprecation procedure at `img-src` and `src` props definition. And without extra variable.

Reply:
something like this

```
dark: {
            // Replaces prop `inverse`
            type: Boolean,
            default() {
                if (this && this.inverse !== null) {
                    // Deprecate inverse
                    warn("b-table: prop 'inverse' has been deprecated. Use 'dark' instead");
                    return this.inverse;
                }
                return false;
            }
        },
        inverse: {
            // Deprecated in v1.0.0.beta.10 in favor of `dark`
            type: Boolean,
            default: null
        },
```